routes/insertRestaurant.js

```diff
@@ -24,37 +24,40 @@ router.get('/', function(req, res, next){
         console.log("Connected correctly to server");
         // create a new dish
         Restaurants.create({
-            name: 'Monty da Dhabba',
-            address:'3800 SW 34th St',
-            city:'Tampa',
-            state:'Florida',
+            name: "Taco House",
+            address: "4295 W Navy Blvd, Pensacola, FL 32507",
+            city: "Pensacola",
+            state: "FL",
             timing:[{
-                open:800,
-                close: 2200
+                open:800,close: 2200
             }],
-            phone:3695556789,
-            expense_level:1,
-            rating:5,
+            phone: 8509124430,
+            expense_level: 3,
+            rating: 3,
             menu:[{
-                name:'Paneer Kadhai',
-                description:'Zindagi ka Pyaar'
-            },
-            {
-                name:'Achari Bhindi Masala',
-                description:'Khushiyon ki Bahaar'
-            },
-            {
-                name:'Lassi',
-                description:'Gale ko rahat'
-            },
-            {
-                name:'Masala Papad',
-                description:'Lassi ke saath aa jaye mazaa'
-            },
-            {
-                name:'Chicken Tikka',
-                description:'Kyun de rhe ho bechare chicken ko sazaa'
-            }]        
+                 name:'Burrito',
+                 description:'Stuffed with meat, cheese, rice, beans and sauce. Then smothered with extra sauce and cheese.'
```

Comment:
we might need a image url in the schema to display images.

Reply:
do we need to add that now itself? Since we aren't planning on storing and working on images

Reply:
I think as of now, we don't require image url